### PR TITLE
Selected tab highlighted

### LINKED ISSUE #131: Selected tab should be highlighted
screen shot 2018-06-13 at 4 20 01 pm
screen shot 2018-06-13 at 4 20 06 pm
When selected leaderboard selected tab is highlighted but when on stats or claims tab the selected item is not highlighted

### LINKED ISSUE #135: navbar doesn't show active tabs properly
should show claim as active when on /claims/* pages and stats active on /stats page

### PR DESCRIPTION
**Pull request by architkshk**
fixes #131 , fixes #135

### CODE REVIEW

**Comment by championswimmer** [comment]
ui changes **must** have screenshots

**Comment by architkshk** [comment]
@championswimmer 
screenshot from 2018-06-15 08-37-28

**Comment by championswimmer** [comment]
please do for claims pages too

check https://github.com/coding-blocks/boss/issues/135

**Comment by architkshk** [comment]
screenshot from 2018-06-15 22-32-43

File: views/layouts/main.hbs [hunk 1]
```diff
@@ -119,7 +123,7 @@
       Leaderboard
     </a>
 
-    <div class="ui simple dropdown item">
+    <div class="ui simple dropdown item {{menu.claims}}">
```
Comment (championswimmer) [inline views/layouts/main.hbs]: `claims-active` isn't a valid class. it should be `active` only

Comment (architkshk) [reply views/layouts/main.hbs]: Adding active class along with drop-down would show drop-down box. So i added claim-active class and defined it in main.hbs

**Pull request merged**